### ISSUE TITLE
Add NamespaceMapper

### ISSUE THREAD

**Issue by wslulciuc** [MarquezProject/marquez#300]
Maps **class** [`NamespaceRequest`](https://github.com/MarquezProject/marquez/blob/master/src/main/java/marquez/api/models/NamespaceRequest.java) to a [`Namespace`](https://github.com/MarquezProject/marquez/blob/master/src/main/java/marquez/service/models/Namespace.java)

**NOTE:** For a great example mapper, see [`DatasetMapper`](https://github.com/MarquezProject/marquez/blob/master/src/main/java/marquez/api/mappers/DatasetMapper.java)

**Comment by wslulciuc** [comment]
Thanks for picking up this issue, @DavidLi119!

**Comment by DavidLi119** [comment]
Moving my questions to here as it is more relevant
I have created a new file called NamespaceMapper.java and made the following changes after copying what was in NamespaceApiMapper.java.
```java
public **final** class NamespaceMapper extends Mapper<NamespaceResponse, Namespace> {
  private NamespaceMapper() {}

  public **static** Namespace map(NamespaceResponse namespace) {
    requireNonNull(namespace, "namespace must not be null");
    return new Namespace(
        null, namespace.getName().toLowerCase(), namespace.getOwner(), namespace.getDescription());
  }
```
 I tried the `./gradlew test` command in VSCode, but it says that it fails because it cannot build due to failing the task `:compileJava` despite me downloading a JDK. Are these changes what you wanted? And how do I test if it works?

**Comment by wslulciuc** [comment]
@DavidLi119, you've made some amazing progress on this issue! Let me know if you need further guidance on writing a test for `NamespaceMapper`.

**Comment by DavidLi119** [comment]
Apologies, I've been working on other stuff at the moment, I will get to it soon.

**Comment by wslulciuc** [comment]
Fixed #488
